foliant/preprocessors/includes.py

```diff
@@ -537,6 +538,20 @@ def _process_include(
         with open(included_file_path, encoding='utf8') as included_file:
             included_content = included_file.read()
 
+            if self.config.get('escape_code', False):
```

Comment:
The only thing that's botheing me is that the preprocessor is called excapecode but the settings is called escape_code. Why?

Reply:
In fact, nevermind, I got it now.